editor/src/core/model/element-metadata-utils.ts

```diff
@@ -1788,6 +1789,44 @@ export const MetadataUtils = {
       isJSXConditionalExpression(element.element.value)
     )
   },
+  findLayoutSystemForChildren(
+    metadata: ElementInstanceMetadataMap,
+    parentPath: ElementPath,
+  ): DetectedLayoutSystem {
+    const children = MetadataUtils.getOrderedChildrenParticipatingInAutoLayout(metadata, parentPath)
```

Comment:
I _think_ we don't need to filter for children that are participating in the layout

Reply:
fixed as part of https://github.com/concrete-utopia/utopia/pull/3334/commits/9061d33830fd6e528efad4f611ca641d1a7ec414